lib/auth/grpcserver_test.go

```diff
@@ -4388,3 +4389,66 @@ func TestDropDBClientCAEvents(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAccessGraphConfig(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{
+		TestFeatures: modules.Features{
+			Policy: modules.PolicyFeature{
+				Enabled: true,
+			},
+		},
+	})
+	server := newTestTLSServer(t,
+		withAccessGraphConfig(AccessGraphConfig{
+			Enabled: true,
+			CA:      []byte("ca"),
+			Address: "addr",
+		}),
+	)
+
+	positiveResponse := &clusterconfigpb.AccessGraphConfig{
+		Enabled: true,
+		Ca:      []byte("ca"),
+		Address: "addr",
+	}
+
+	tests := []struct {
+		desc      string
+		role      types.SystemRole
+		assertErr require.ErrorAssertionFunc
+		expected  *clusterconfigpb.AccessGraphConfig
+	}{
+		{
+			desc:      "admin role can't pull the access graph config",
+			role:      types.RoleAdmin,
```

Comment:
I don't think this makes sense, as the admin can still pull the resources from the backend anyway.